src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/github/v4/GraphqlCheckRunProvider.java

```diff
@@ -104,6 +106,7 @@ public void createCheckRun(AnalysisDetails analysisDetails, UnifyConfiguration u
         headers.put("Accept", "application/vnd.github.antiope-preview+json");
 
         List<InputObject<Object>> annotations = analysisDetails.getPostAnalysisIssueVisitor().getIssues().stream()
+                .limit(GITHUB_CHECKS_ANNOTATIONS_MAX_NUMBER_PER_REQUEST)
```

Comment:
Should we not be making multiple calls to the endpoint to submit all the annotations, rather than just submitting a subset? At the very least, I'd expect us to warn the user we've only submitted a subset of the findings